easybuild/easyconfigs/e/ExaML/ExaML-3.0.21-intel-2018a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'MakeCp'
+
+name = 'ExaML'
+version = '3.0.21'
+
+homepage = 'https://cme.h-its.org/exelixis/web/software/examl/index.html'
+
+description = """Exascale Maximum Likelihood (ExaML) code for phylogenetic inference on supercomputers using MPI.
+This code implements the popular RAxML search algorithm for maximum likelihood based inference of phylogenetic trees.
+It uses a radically new MPI parallelization approach that yields improved parallel efficiency, in particular on
+partitioned multi-gene or whole-genome datasets. It is up to 3.2 times faster than RAxML-Light. As RAxML-Light, 
+ExaML also implements checkpointing, SSE3, AVX vectorization and memory saving techniques."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = ['https://github.com/stamatak/ExaML/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['6c7e6c5d7bf4ab5cfbac5cc0d577885272a803c142e06b531693a6a589102e2e']
+
+prebuildopts = 'cd examl && '
+buildopts = '-f Makefile.OMP.AVX.gcc'  # This makefile actually uses mpicc
```

Comment:
Usually we prefer to patch the Makefile that have hardcoded compilers and flags so that they take the value set by EasyBuild in `CC`, `MPICC`, `CFLAGS`, etc.
Would this be possible here?

Reply:
Maybe we should just slap `versionsuffix` like `-OMP-AVX` on it?

Reply:
That would be fine by me as well (I considered doing so at first)

Reply:
I agree on @boegel suggestion. You will still need a patch for the Makefile to use the correct compiler and flags.

Reply:
@Micket ping on using a `versionsuffix` here?

Reply:
Sorry, been busy. I would be fine with a suffix, but I haven't had time to look into vanzod's comment. It already uses the right compiler as far as I know. I intentionally avoided fiddling with the flags, as it already hardcodes AVX intrinsics in the code.